README.md

```diff
@@ -26,7 +26,6 @@ After adding the name execute update.py to update the readme.
 * Avinash Agarwal [manu4rhyme](https://github.com/manu4rhyme)
 * Dushyant Mishra [zero1hac](https://github.com/zero1hac)
 * Govind Sahai [mafiya69](https://github.com/mafiya69)
-* Harshit [iamharshit] (https://github.com/iamharshit)
```

Comment:
No, I guess his name was never added to AUTHORS.md. If you add there and run the script, it should be good.